charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -40,6 +40,18 @@ spec:
       resources:
         requests:
           storage: {{ .Values.cassandra.cassandraLibDirVolume.size | default "5Gi" }}
+    {{- if and (eq .Values.medusa.storage "local") ( .Values.medusa.podStorage ) }}
+    additionalVolumes:
+      - mountPath: /mnt/backups
+        name: medusa-backups
+        pvcSpec:
```

Comment:
If we choose not to use default values for `podStorage.size` and `podStorage.storageClass`, can we check and fail rendering if they are not set when storage is `local`? 

Note that we do use default values for the `.spec.storageConfig` property.